.yardopts

```diff
@@ -0,0 +1,5 @@
+--markup markdown
```

Comment:
@keithdoggett this changes things a bit in fact: it favors markdown in stead of rdoc. Hence the next code:

```
# this is `markdown` versus +rdoc+
```

Would render the `markdown` part but not the +rdoc+ part. I don't think we can use both. And I'm leaning towards markdown since it is more powerful and well known. What is your opinion on that ?

Reply:
So this means that it will not generate documentation based on the comments in the code?

Reply:
@keithdoggett yes it will ! But it will use the markdown syntax rather than the rdoc syntax. Yardoc specific syntax will still be available either way

Reply:
Ok, so this means that we will have to change the RDoc specific syntax to markdown syntax.

For example in this snippet:
``` 
# [<tt>:has_z_coordinate</tt>]
#   Set to true if geometries created by this factory include a Z
#   coordinate, and the Point#z method is available.
# [<tt>:has_m_coordinate</tt>]
#   Set to true if geometries created by this factory include a M
#   coordinate, and the Point#m method is available.
# [<tt>:is_cartesian</tt>]
#   Set to true if this Factory guarantees that it operates in
#   Cartesian geometry. If false or nil, no such guarantee is made,
#   though it is possible the geometries may still be Cartesian.
# [<tt>:is_geographic</tt>]
#   Set to true if this Factory's coordinate system is meant to be
#   interpreted as x=longitude and y=latitude. If false or nil, no
#   information is present about whether the coordinate system is
#   meant to be so interpreted.
```
The `<tt>` tags would have to be changed to the equivalent in markdown?

Reply:
This is kind of wrong any way and would need to use rather yard specific syntax. However i don't know how does this render using the yardopts we added. Did you take a look at that part of the doc ?

Reply:
Looks like it works, but it probably needs an extra line between each option. It's rendering everything on the same line for me. Although that might be an issue with my setup since I was seeing the same thing with the table.